### PR TITLE
More trivial code cleanup

### PR DESCRIPTION
**Pull request by dedmen**
Related to #6858
pboprefixes are for both compile and for armake (as armake can also scan for pboprefix, atleast armake++)

### CODE REVIEW

File: addons/common/functions/fnc_checkFiles.sqf [hunk 1]
```diff
@@ -69,8 +69,8 @@ if (!(_oldCompats isEqualTo [])) then {
 ///////////////
 // check dlls
 ///////////////
-if (toLower (productVersion select 6) in ["linux", "osx"]) then {
-    INFO_2("Operating system does not support DLL file format");
```
Comment (dedmen) [inline addons/common/functions/fnc_checkFiles.sqf]: How did this not get noticed before? and why didn't arma crash at this invalid macro usage? and why did our linter not pick that up?
so many questions.

**Comment by dedmen** [comment]
defineResincl.inc required by map/functions/fnc_initMainMap.sqf

File: addons/common/functions/fnc_checkFiles.sqf (hunk 1 above)
Comment (bux) [reply addons/common/functions/fnc_checkFiles.sqf]: per se linux doesn't support *.dll files, does ist?

**Comment by dedmen** [comment]
defineResinclDesign.inc
optionsmenu/init_loadMainMenu.sqf

**Comment by dedmen** [comment]
All typos/bugs/errors detected by my script profiler.
We should run that on CI.
I'll add a argument to recursively compile a folder with multithreading. Should only take a couple seconds for whole ace.

File: addons/common/functions/fnc_checkFiles.sqf (hunk 1 above)
Comment (PabstMirror) [reply addons/common/functions/fnc_checkFiles.sqf]: none of our releases have ever included `.so` files
with this line, people on linux can at least attempt to use all ace features that do not rely on extensions
now it will block access entirely

Comment (dedmen) [reply addons/common/functions/fnc_checkFiles.sqf]: I thought I have seen them on linux server :thinking:

**Comment by dedmen** [comment]
SQF-VM can now compile all of ace without errors, besides one file in zeus that's really just a redirect to a vanilla file, because the include is not available. And I won't copy that script file into our includes as it doesn't make sense.

File: include/a3/data_f/$PBOPREFIX$ [hunk 2]
```diff
@@ -0,0 +1 @@
+a3\data_f
```
Comment (jonpas) [inline include/a3/data_f/$PBOPREFIX$]: Why are those necessary? Those files are already in folders representing prefix, and armake doesn't need them.

Comment (dedmen) [reply include/a3/data_f/$PBOPREFIX$]: Because some tools (compiler) search for pboprefix to be sure that it's what it should be.
Otherwise the prefix might aswell be include\a3\data_f or ace\include\a3\data_f or whateve

Comment (TheMagnetar) [reply include/a3/data_f/$PBOPREFIX$]: EOF?

Comment (jonpas) [reply include/a3/data_f/$PBOPREFIX$]: Can be present, any normal tool will parse it just fine.

Comment (dedmen) [reply include/a3/data_f/$PBOPREFIX$]: Some bad tools just take the whole $PBOPREFIX$ as prefix.
So if you pack with these you end up with prefix\n 
soo stupid.. 
There is no reason to have newline at the end here really anyway.

Comment (jonpas) [reply include/a3/data_f/$PBOPREFIX$]: Forget those tools, just leave it as is. ;)

Comment (dedmen) [reply include/a3/data_f/$PBOPREFIX$]: cpbo is one of them. And I use that to build pbo's to quickly test things :D

Comment (jonpas) [reply include/a3/data_f/$PBOPREFIX$]: Judging.

Comment (dedmen) [reply include/a3/data_f/$PBOPREFIX$]: Solution would be to write a script that let's you add armake pbo packing to context menu.
Context menu stuff is a chore to do though.
